.github/workflows/ci/pkg/bot/check.go

```diff
@@ -57,6 +57,15 @@ func (c *Bot) checkInternal(ctx context.Context) error {
 	if err != nil {
 		return trace.Wrap(err)
 	}
+	// If @r0mant and @russjones approve a pull request, allow the check workflow run to pass.
+	// This is used in the event there is someone out of the office and the pull request needs the bypassing
+	// of the required reviewers.
+	if len(ci.RepoAdmins) == 2 {
+		if hasApproved(ci.RepoAdmins[0], mostRecentReviews) && hasApproved(ci.RepoAdmins[1], mostRecentReviews) {
+			log.Println("Admins @r0mant and @russjones have approved.")
```

Comment:
Might be good to use the `RepoAdmins` var here rather than hardcoding the usernames